### PR TITLE
Dapolloxp/models

### PR DESCRIPTION
**Pull request by dapolloxp**
- Modified Visualize File to be able to leverage Shawn's Data files

- Added Scratch NB to test SVR (Support Vector Regression)

### CODE REVIEW

**Review by tedsmoore — CHANGES_REQUESTED** [review]
Changes look good and nice to have a scratch regression notebook. Can you just add .idea to the .gitignore file?

File: src/visualization/visualize.py [hunk 1]
```diff
@@ -24,14 +23,14 @@ def graph_stock(self):
         
         plt.xlabel("Date")
         plt.ylabel("Price")
-        stock_open_price = pd.Series(self._ticker['Open'].to_numpy(), index=pd.date_range(self._ticker['Date'].iloc[0], periods=503, freq='D'))
-        stock_close_price = pd.Series(self._ticker['Close'].to_numpy(), index=pd.date_range(self._ticker['Date'].iloc[0], periods=503, freq='D'))
+        stock_open_price = pd.Series(self._ticker['Open'].to_numpy(), index=pd.date_range(self._ticker['Date'].iloc[0], periods=len(self._ticker), freq='D'))
```
Comment (tedsmoore) [inline src/visualization/visualize.py:26]: Use `freq='B'` for business days. Also, there's an parameter for `holidays` to get our dates lined up exactly.

Comment (dapolloxp) [reply src/visualization/visualize.py:26]: removed pyalgo

**Comment by tedsmoore** [comment]
Might still need to run a command to clear out this .idea folder from git history. I think it's:

```git rm -r --cached .idea```

Sorry for being a pain about that but I'm afraid it'll screw with my PyCharm settings.

**Review by tedsmoore — APPROVED** [review]
Approving so you can merge, but please rm the .idea folder first 👍

**Pull request merged**